homeassistant/components/stream/worker.py

```diff
@@ -60,32 +43,133 @@ def stream_worker(hass, stream, quit_event):
     except (KeyError, IndexError):
         _LOGGER.error("Stream has no video")
         return
+    try:
+        audio_stream = container.streams.audio[0]
```

Comment:
Should we check here if the audio stream matches one of our acceptable formats and just set to `False` if not? That would save us from remuxing feeds with G.711 streams that won't be playable.

Reply:
Nevermind, I see what's happening now.